src/Microsoft.AspNetCore.Sockets.Common/Internal/Formatters/MessageFormatUtils.cs

```diff
@@ -13,11 +14,11 @@ public static byte[] DecodePayload(byte[] inputPayload)
             if (inputPayload.Length > 0)
             {
                 // Determine the output size
-                var decodedLength = Base64Encoder.ComputeDecodedLength(inputPayload);
+                var decodedLength = Base64Decoder.ComputeDecodedLength(inputPayload);
 
                 // Allocate a new buffer to decode to
                 var decodeBuffer = new byte[decodedLength];
-                if (!Base64Encoder.TryDecode(inputPayload, decodeBuffer, out _, out var _))
+                if (Base64.Decoder.Transform(inputPayload, decodeBuffer, out _, out var _) != TransformationStatus.Done)
```

Comment:
I think it is fine to merge like this to unblock the mirror but then make sure that we don't need to handle other cases. If we do, we will do it separately.